README.md

```diff
@@ -53,6 +53,8 @@ By default, `@NonBlocking` macro will only report violations in `DEBUG` configur
 func process() { print("") }
 ```
 
+On Linux, make sure to call `RealtimeSanitizer.ensureInitialized()` early in your process lifecycle.
```

Comment:
I would recommend just saying "Make sure to call `RealtimeSanitizer.ensureInitialized()` early in your process lifecycle"

It does not do any harm on MacOS, and people may think they have to do some #ifdef special inclusion.

Up to you though, just a suggestion